src/aws_iot_shadow.c

```diff
@@ -33,10 +33,10 @@ extern "C" {
 #include "aws_iot_shadow_records.h"
 
 const ShadowInitParameters_t ShadowInitParametersDefault = {(char *) AWS_IOT_MQTT_HOST, AWS_IOT_MQTT_PORT, NULL, NULL,
-															NULL, false, NULL};
+															NULL, false};
```

Comment:
I think there should be a NULL after the false here to initialize `ShadowInitParametersDefault.disconnectHandler`.  Can you take a look at this?

Reply:
I cannot speak for @pctj101 but I think the `NULL` is necessary.